tests/unit/helpers/customers.test.js

```diff
@@ -765,6 +766,35 @@ describe('updateCustomer', () => {
       ]
     );
   });
+
+  it('should update status', async () => {
```

Comment:
On pourrait fusionner ce test avec celui au dessus (should update a customer), qu'en penses-tu ?

Reply:
je pense ce teste ne sert pas 🤔